app/Makefile.version

```diff
@@ -1,6 +1,6 @@
 # This is the `transaction_version` field of `Runtime`
-APPVERSION_M=0
+APPVERSION_M=1
```

Comment:
last published version from Lisk app was 2.0.3. We might need to set a higher version before sending this to Ledger